easybuild/tools/build_log.py

```diff
@@ -52,7 +53,7 @@
 DEPRECATED_DOC_URL = 'http://easybuild.readthedocs.org/en/latest/Deprecated-functionality.html'
 
 
-class EasyBuildError(Exception):
+class EasyBuildError(LoggedException):
```

Comment:
you will have to redefine the error logger here to a pure fancylogger (ortherwise error will go in some loop). use a function that sets raiseError to false, logs error and set it back to true (like exception does)